### PR TITLE
Use https

### PR DESCRIPTION
**Pull request by aidenkeating**
Use https instead of http  

Verification steps:
* Provision the Keycloak service
* Ensure the route is https
* Ensure Keycloak is accessible
* Ensure bind and unbind continue to work correctly

### CODE REVIEW

**Comment by aidenkeating** [comment]
@rachael-oregan Would you mind taking a look?

**Comment by wei-lee** [comment]
👀

File: roles/bind-keycloak-apb/tasks/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 - name: "Generate keycloak auth token"
   uri:
-    url: "http://{{ keycloak_route.stdout }}/auth/realms/master/protocol/openid-connect/token"
+    url: "https://{{ keycloak_route.stdout }}/auth/realms/master/protocol/openid-connect/token"
```
Comment (wei-lee) [inline roles/bind-keycloak-apb/tasks/main.yml]: can we make the protocol value a parameter and default it to `https`? I do think there might be a use case of http for testing purpose only.

@aidenkeating WDYT?

Comment (aidenkeating) [reply roles/bind-keycloak-apb/tasks/main.yml]: @wei-lee Yeah that's a good point 👍

**Comment by rachael-oregan** [comment]
@aidenkeating sorry only this now. LGTM.  Ping once Wei's comment is addressed and I'll start the verification

**Comment by aidenkeating** [comment]
@wei-lee @rachael-oregan There's now a `keycloak_protocol` variable in `vars/main.yml`. It defaults to `https`.

**Comment by rachael-oregan** [comment]
@aidenkeating I'll start the verification now

**Comment by rachael-oregan** [comment]
@aidenkeating verified

**Comment by aidenkeating** [comment]
@rachael-oregan Awesome, thanks!

@wei-lee I think the `https/http` change covers everything in that comment. Happy for this to be merged?

**Comment by wei-lee** [comment]
@aidenkeating 👍

**Pull request merged**